### PR TITLE
Docs for PR 9373

### PR DESCRIPTION
**Pull request by jkrumbiegel**
Describes usage of `QUARTO_JULIA_PROJECT` added in https://github.com/quarto-dev/quarto-cli/pull/9373

(Edit from @cwickham): [Preview](https://deploy-preview-1078--mystifying-jepsen-fa4396.netlify.app/docs/computations/julia.html#installation-1)

### CODE REVIEW

File: docs/computations/julia.qmd [hunk 1]
```diff
@@ -210,6 +210,10 @@ To enable the cache for a document, add the `cache` option. For example:
...
 
+#### Using custom versions of `QuartoNotebookRunner`
+
+In special circumstances, you may not want to use the specific `QuartoNotebookRunner` version that quarto installs for you. One reason could be that you're developing `QuartoNotebookRunner` itself, or you need to use a fork or an unreleased version with a bugfix. In this case, you can run quarto with the environment variable `QUARTO_JULIA_PROJECT` set to a directory of a julia environment that has `QuartoNotebookRunner` installed. As an example, you could install the main branch of `QuartoNotebookRunner` into the directory `/some/dir` by executing `]activate /some/dir` in a julia REPL and then `]add QuartoNotebookRunner#main`. If there is no server process running, yet, after running a command like `QUARTO_JULIA_PROJECT=/some/dir quarto render some_notebook.qmd` in your terminal, the server process should be started using the custom `QuartoNotebookRunner`. You can also set quarto's `--execute-debug` flag and check the output to verify that the custom environment is being used.
```
Comment (cwickham) [inline docs/computations/julia.qmd:215]: A few wording suggestions:

```suggestion
In special circumstances, you may not want to use the specific `QuartoNotebookRunner` version that Quarto installs for you. For example, you might be developing `QuartoNotebookRunner` itself, or you need to use a fork or an unreleased version with a bugfix. In this case, set the [environment variable](/docs/projects/environment.qmd) `QUARTO_JULIA_PROJECT` to a directory of a julia environment that has `QuartoNotebookRunner` installed. 

As an example, you could install the main branch of `QuartoNotebookRunner` into the directory `/some/dir` by executing `]activate /some/dir` in a julia REPL followed by `]add QuartoNotebookRunner#main`. As long as there is no server currently running, running a command like `QUARTO_JULIA_PROJECT=/some/dir quarto render some_notebook.qmd` in your terminal will ensure the server process is started using the custom `QuartoNotebookRunner`. You can also set `quarto`'s `--execute-debug` flag and check the output to verify that the custom environment is being used.
```

I added a link to https://quarto.org/docs/projects/environment.html ... it seemed relevant here.

**Review by cwickham — APPROVED** [review]
Thanks @jkrumbiegel ! Looks good. I made some wording suggestions. If you are OK with them, commit them, and I'll take one last look at the preview before merging.

**Comment by jkrumbiegel** [comment]
Thank you, I have applied your suggestions :)

**Pull request merged**